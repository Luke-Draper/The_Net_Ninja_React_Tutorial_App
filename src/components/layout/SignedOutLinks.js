import React, { Component } from "react";

class SignedOutLinks extends Component {
	render() {
		return (
			<div className="signed-out-links">
				<h3>SignedOutLinks</h3>
			</div>
		);
	}
}

export default SignedOutLinks;
