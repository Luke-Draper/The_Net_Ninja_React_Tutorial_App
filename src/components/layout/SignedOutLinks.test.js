import React from "react";
import SignedOutLinks from "./SignedOutLinks";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("SignedOutLinks renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<SignedOutLinks />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
