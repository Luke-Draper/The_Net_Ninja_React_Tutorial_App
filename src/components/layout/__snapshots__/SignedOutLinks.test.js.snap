// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedOutLinks renders without crashing 1`] = `
<ul
  className="right SignedOutLinks"
>
  <li>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Sign Up
    </a>
  </li>
  <li>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Log In
    </a>
  </li>
</ul>
`;
